src/java/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerStandard.java

```diff
@@ -58,6 +60,10 @@ public void discard(CommitLogSegment segment, boolean delete)
             segment = allocatingFrom();
         }
 
+        long elapsed = ApproximateTime.currentTimeMillis() - start;
+        if (elapsed >= ApproximateTime.precision())
+            noSpamLogger.info("Took {} ms to get a commit log allocation", elapsed);
```

Comment:
@ekaterinadimitrova2 Question: What is a "longer than the precision of the timer" elapsed time typically caused by?